### DISCUSSION TITLE
Can I use WatermelonDB with Cordova/Capacitor?

### DISCUSSION THREAD

**Discussion by a777med** [Nozbe/WatermelonDB#1088] [Q&A]
Hello,

I would like to use WatermelonDB with a webview app. Like, Cordova or Capacitor. Is that possible?

Thanks.

**Comment by isaachinman** [comment]
Three years later – any answer?

I see a mention of a Capacitor/sqlite adaptor [here](https://watermelondb.dev/docs/Implementation/DatabaseAdapters#contribute-these-adapters).
